web/middlewares.go

```diff
@@ -17,7 +17,7 @@ import (
 )
 
 // limiter middleware pointer
-var limiterMiddleware *stdlib.Middleware
+var LimiterMiddleware *stdlib.Middleware
```

Comment:
In go, each exposed variables should be doc string starting with such name, i.e. in this case you should also change doc string to
```
// LimiterMiddleware provides limiter middleware pointer
var LimitedMIddleware
```

And, all non-exposed variables do not have this constrain